### PR TITLE
Adds a simple 'persistence' subsystem

### PR DESCRIPTION
**Pull request by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE**
Something other servers have had for a while now.
The proof of concept is a simple "round counter". It just tracks how many rounds have been played to completion each year. The end credits now select the season and episode number as SEASON [last 2 digits of current IRL year] EPISODE [amount of rounds played so far this year]

### CODE REVIEW

File: data/persistence/round_counts_per_year.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"19":"409"}
```
Comment (I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE) [inline data/persistence/round_counts_per_year.json]: Actually accurate at the time of writing this.

**Comment by Irradiation** [comment]
Gob dless

File: data/persistence/round_counts_per_year.json (hunk 1 above)
Comment (Irradiation) [reply data/persistence/round_counts_per_year.json]: If Pomf is to be believed, server started in May 2013, SO WHAT HAPPENED TO SEASONS 1 TO 12, HUH?

Comment (I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE) [reply data/persistence/round_counts_per_year.json]: Pomf accidentally overwrote the VHS tapes

Comment (Irradiation) [reply data/persistence/round_counts_per_year.json]: Lynch Pomf when?

**Comment by Kurfursten** [comment]
Too powerful for mortal coders

File: data/persistence/round_counts_per_year.json (hunk 1 above)
Comment (d3athrow) [reply data/persistence/round_counts_per_year.json:1]: Why wouldn't pomf be believed?

Comment (d3athrow) [reply data/persistence/round_counts_per_year.json:1]: Also we've had 29865 rounds since may 2013

Comment (I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE) [reply data/persistence/round_counts_per_year.json]: in the year
29865

Comment (d3athrow) [reply data/persistence/round_counts_per_year.json:1]: just for fun here's the monthly round count since i started hosting
asdf

Comment (Exxion) [reply data/persistence/round_counts_per_year.json:1]: What happened in late 2014?

Comment (Kurfursten) [reply data/persistence/round_counts_per_year.json:1]: November 2014 Badclown Hosting Incident

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
Actually wait, how will the repo react to me force-comitting a .gitignored file? Will it be a dummy part of the repo forever?

**Comment by d3athrow** [comment]
i have no idea

**Comment by Exxion** [comment]
You forget that nobody here *actually* knows how Git works

**Comment by SonixApache** [comment]
>`>>243266743`
`An idea for shift-to-shift continuity: Beepsky's name should be whatever the name of the last named securitron was.`

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
I don't really like the idea of destroying Beepsky's legacy like that tbh

**Pull request merged**